Source/control.h

```diff
@@ -83,8 +84,8 @@ inline bool CanPanelsCoverView()
 	return GetScreenWidth() <= mainPanel.size.width && GetScreenHeight() <= SidePanelSize.height + mainPanel.size.height;
 }
 
-void AddInfoBoxString(std::string_view str);
-void AddInfoBoxString(std::string &&str);
+void AddInfoBoxString(std::string_view str, bool floatingBox = false);
+void AddInfoBoxString(std::string &&str, bool floatingBox = false);

```

Comment:
This to me feels a little odd, but I'm not a big fan of the `AddInfoBoxString()` to begin with either.